cwarhm/data_specific_processing/era5.py

```diff
@@ -0,0 +1,227 @@
+
```

Comment:
general comment: this whole function could probably be done much easier with xarray, including the renaming of variables.

Reply:
Agree. Code improvement can be started after conversion is done.